.gitignore

```diff
@@ -1,3 +1,4 @@
 target/
 project/project/
 project/target/
+.idea
```

Comment:
Actually, this is considered a bad practice. Such things as IDE's, dev-env, OS, etc. specific files should be placed into `.gitignore_global`. The motivation behind it - it easy becomes a nightmare to support all the files for all the possible IDEs etc.
Library gitignore file should contain files related only to the library itself and language/platform it's written in.

Reply:
I see. I am happy if your remove it.